make/tools.mk

```diff
@@ -370,6 +371,19 @@ $(BINDIR)/downloaded/tools/ko@$(KO_VERSION)_%: | $(BINDIR)/downloaded/tools
 	chmod +x $@
 	rm $@.tar.gz
 
+###############
+# boilersuite #
+###############
+
+BOILERSUITE_linux_amd64_SHA256SUM=33be81a0d7b4b9a0039730d4b4f22af272d7c7586c685ee5f8819f1dbefc0336
+BOILERSUITE_darwin_amd64_SHA256SUM=b6ee2355a6087c4e480caf8eecd3463f710dc8c13e100f5f4d433fea8e1ccc4a
+BOILERSUITE_darwin_arm64_SHA256SUM=883781c17a6958715515ad41225cbbfa05d0273bc4f5b309cb78b7e265c68552
+
+$(BINDIR)/downloaded/tools/boilersuite@$(BOILERSUITE_VERSION)_%: | $(BINDIR)/downloaded/tools $(BINDIR)/tools
+	$(CURL) https://github.com/cert-manager/boilersuite/releases/download/$(BOILERSUITE_VERSION)/boilersuite-$(subst _,-,$*)  -o $@
+	./hack/util/checkhash.sh $@ $(BOILERSUITE_$*_SHA256SUM)
+	chmod +x $@
+
```

Comment:
Can we download boilersuite using go instead (that way we don't have to keep track of the sha256)?
```
GO_DEPENDENCIES += boilersuite=github.com/cert-manager/boilersuite
```

Reply:
Ahh when you mentioned about the binaries yesterday I assumed you wanted it installed that way with cert-manager; obviously makes sense to do it via Go here since we can rely on Golang being available in the environment. Will change!

Reply:
Sorry for the confusion, I was thinking about non-go projects (eg. a configuration repo).